forms-web-app/src/controllers/eligibility.js

```diff
@@ -10,7 +10,7 @@ exports.getDecisionDate = (req, res) => {
 
 exports.postDecisionDate = (req, res) => {
   const { body } = req;
-  const { errors } = body;
+  const errors = body.errors || [];
```

Comment:
If this is to default to an array, you can write this as `const { errors = [] } = body;`

Reply:
ah, smashing, ta

Reply:
No probs. I've no issue with the way you did it - it's perfectly valid - but that's just an FYI for how to do it with destructuring